frontend/src/components/general/Form.tsx

```diff
@@ -249,6 +251,7 @@ export default function Form({
                 <SelectField
                   controlledValue={{ name: values[field.key] }}
                   controlled
+                  // @ts-ignore - contrast is a custom color defined in theme
                   color="contrast"
```

Comment:
Strange that line 275 with the same custom color did not need this annotation. SelectField is actually our own component using: `color?: any`